gensim/models/keyedvectors.py

```diff
@@ -191,6 +191,8 @@
 
 KEY_TYPES = (str, int, np.integer)
 
+_EXTENDED_KEY_TYPES = (str, int, np.integer, np.ndarray)
+
```

Comment:
This doesn't match the style of the immediately-preceding module constant: shouldn't they both be underscore-preceded, or neither? (And: I prefer to be sparing with leading-underscores, only where there's a strong implication of "internal only".)

Reply:
OK, I've made them both internal.